web/src/user_pill.ts

```diff
@@ -84,6 +86,7 @@ export function create_item_from_email(
         deactivated: false,
         status_emoji_info,
         should_add_guest_user_indicator: people.should_add_guest_user_indicator(user.user_id),
+        user_is_bot: user_is_bot(user.user_id),
```

Comment:
Can't one check `user.id_bot` directly? I don't see why you'd need a new function for this.